CONTRIBUTING.md

```diff
@@ -308,39 +308,57 @@ You can find an installation guide for all the different environments here:
+- **📋 Sprint Backlog**: The issue is scheduled for the current sprint.
+- **🏗 In progress**: Someone is working on this issue right now.
+- **👀 In review**: The issue is in review.
+- **✅ Done**: The issue is implemented and merged to main.
 
 ### Priority
 
 Priority shows you the priority the ZITADEL team has given this issue. In general the more customers want a feature the higher the priority gets.
 
-- **priority: critical**: This is a security issue or something that has to be fixed urgently, because customers can't work anymore.
-- **priority: high**: These are the issues the ZITADEL Team is currently focusing on and will be implemented as soon as possible.
-- **priority: medium**: After all the high issues are done these will be next.
-- **priority: low**: This is low in priority and will probably not be implemented in the next time or just if someone has some time in between.
+- **🌋 Critical**: This is a security issue or something that has to be fixed urgently, because customers can't work anymore.
+- **🏔 High**: These are the issues the ZITADEL Team is currently focusing on and will be implemented as soon as possible.
+- **🏕 Medium**: After all the high issues are done these will be next.
+- **🏝 Low**: This is low in priority and will probably not be implemented in the next time or just if someone has some time in between.
 
-### State
 
-The state should reflect the progress of the issue and what is going on right now.
+### Complexity
+
+This should give you an indication how complex the issue is. It's not about the hours or effort it takes.
+Everything that is higher than 8 should be split in smaller parts.
 
-- **state: triage**: Each issue gets this state automatically on creating and it means the ZITADEL team should have a look at it, prioritize and sort into categories or ask for more information if needed.
-- **state: tbd**: If the issue has the state tbd (to be defined) it means the team does need more information either from the author or internal.
-- **state: backlog**: If an issue is in the backlog, it is not currently being worked on. These are recorded so that they can be processed in the future. Issues with this state do not have to be completely defined yet.
-- **state: ready**: An issue with the state ready is ready to implement. This means the developer can find all the relevant information and acceptance criteria in the issue.
-- **state: in progress**: Someone is working on this issue right now.
-- **state: waiting**: For some reason, this issue will have to wait. This can be a feedback that is being waited for, a dependent issue or anything else.
-- **state: duplicate**: The same issue already exists. This issue will probably be closed with a reference to the other issue.
+**1**, **2**, **3**, **5**, **8**, **13**
+
+### Sprint
+
+We want to deliver a new release every second week. So we plan everything in two week sprint.
```

Comment:
As mentioned above: Maybe add start / end day and planned release periodicity of the sprints.

Reply:
I have changed the structure. Not sure why I placed this part here, makes more sense at the top